internal/resources/webform/webform.js

```diff
@@ -154,9 +154,6 @@ window.initGRPCForm = function(services, svcDescs, mtdDescs, invokeURI, metadata
         // set raw request text
         updateJSONRequest(requestObj);
 
-        // init grpcCurl text
-        updateCurlCommand(requestObj);
-
```

Comment:
this part actually seems fine since `updateJSONRequest` calls this internally

Reply:
@dragonsinth  can you clarify please. The call to `updateCurlCommand ` is not required here, since `updateJSONRequest` will do that.   So removing this call actually fixes the issue.